pages/common/crontab.md

```diff
@@ -13,3 +13,19 @@
 - Remove all cron jobs for the current user:
 
 `crontab -r`
+
+- Cron job definition format:
+
+`{{minute}} {{hour}} {{day_of_month}} {{month}} {{day_of_week}} {{command_to_execute}}`
```

Comment:
How about adding this as a second line in the main description? (without the heavy token marks, maybe parenthesis or brackets)